### DISCUSSION TITLE
"invalid frame header" upon Disconnect

### DISCUSSION THREAD

**Discussion by nagamoto03** [miguelgrinberg/Flask-SocketIO#2076] [Q&A] [answered]
I executed the example code from the repository.

I accessed http://127.0.0.1:5000/ in Chrome and checked the communication via the network tab in the Developer Tools.

After pressing the disconnect button, it showed "invalid frame header".

Furthermore, the console displayed the following error message:
"WebSocket connection to 'ws://127.0.0.1:5000/socket.io/?EIO=4&transport=websocket&sid=np_-TfOQ8qP9GSpIAAAG' failed: Invalid frame header"

Why does this error message appear?

It should be noted that this issue only occurs in Chrome and does not occur in Firefox or Safari.

I tried under various conditions but couldn't figure out the cause, so I am asking for help.

**Comment by miguelgrinberg** [comment, marked as answer]
Which web server are you using? If you are using Werkzeug, I suggest you try with Gunicorn instead, which has better support for WebSocket.
